src/test/java/com/hocs/test/pages/managementUI/ListsManagement.java

```diff
@@ -4,14 +4,18 @@
 import static net.serenitybdd.core.Serenity.setSessionVariable;
 
 import com.hocs.test.pages.decs.BasePage;
+import com.hocs.test.pages.decs.Documents;
 import net.serenitybdd.core.pages.WebElementFacade;
 import org.junit.Assert;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.support.FindBy;
 
 public class ListsManagement extends BasePage {
 
     MUIDashboard MUIDashboard;
 
+    Documents documents;
```

Comment:
Might just be being bling, but I cant see this Documents import being used at all?